basin3d/synthesis.py

```diff
@@ -706,7 +684,8 @@ def _output_df(output_directory, output_name, query_info, metadata_store, first_
                                 query=query_info)
 
 
-def _output_hdf(output_directory: str, output_name: str, query_info: QueryInfo, metadata_store: dict, first_timestamp, last_timestamp,
+def _output_hdf(output_directory: str, output_name: str, query_info: QueryInfo, metadata_store: dict, first_timestamp,
+                last_timestamp,
```

Comment:
minor: weird line break?

Reply:
This is actually formatting to keep the line length reasonable.